docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -1169,6 +1169,12 @@ Wave II
     - Baseline
     - 
     - `Code to generte mean draw for all artifact keys except the LBWSG PAF can be found here <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/mean_draw_generation.ipynb>`_. The mean LBWSG PAF can be calculated using the LBWSG PAF calculation code using the mean draw for LBWSG RRs and LBWSG exposure.
+  * - 13
+    - Production runs using model version 12.1.1/branch bugfix/MIC-4769_mms_bug
```

Comment:
This will be merged into main soon so we can remove the reference to this branch :)